docs/DeveloperGuide.adoc

```diff
@@ -279,13 +279,14 @@ Refer to the guide <<DevOps#, here>>.
 
 *Target user profile*:
 
-* has a need to manage a significant number of contacts
+* health conscious individuals
+* prefers to have a complete health monitoring all in one app
 * prefer desktop apps over other types
 * can type fast
 * prefers typing over mouse input
 * is reasonably comfortable using CLI apps
 
-*Value proposition*: manage contacts faster than a typical mouse/GUI driven app
+*Value proposition*: monitors all aspects of health in one application
 
 [appendix]
 == User Stories
```

Comment:
Have you considered adding more user stories? Could consider including additional stories that would be possible in v2.0